.vscode/settings.json

```diff
@@ -26,7 +26,7 @@
     },
   ],
   "java.test.defaultConfig": "WPIlibUnitTests",
-  "editor.tabSize": 8,
+  "editor.tabSize": 4,
```

Comment:
Don't change the value of a tab. A bad practice and just masks the incorrect indentation.